pkg/operation/garden/garden.go

```diff
@@ -213,6 +229,12 @@ func ReadGardenSecrets(k8sInformers kubeinformers.SharedInformerFactory) (map[st
 		return nil, fmt.Errorf("can only accept at most one OpenVPN Diffie Hellman secret, but found %d", numberOfOpenVPNDiffieHellmanSecrets)
 	}
 
+	// Operators can configure gardener to send email alerts or send the alerts to an external alertmanager. If no configuration
+	// is provided then no alerts will be sent.
+	if numberOfAlertingSecrets > 1 {
```

Comment:
Did we now decide to use either the internal alertmanager with email alerting OR an external alertmanager where you can use what ever you want?

The option to use both in parallel is not in scope anymore?  Because this would need to allow more then one alerting secret.

Reply:
So allowing both will make it difficult to verify having at up to 1 SMTP secret and up to 1 External alertmanager since they are all configured in the same secret. Of course it is still possible but after trying to implement it I find this solution much cleaner.

Reply:
Yes in one secret this would be probably a little difficult, but this wasn't my intention. I suggested to have one alerting secret for each alert config. This would allow to have multiple secrets, but they are all structured in the same way (and the old secret get replaced with a new one).

Anyway I'm totally fine with the either internal alertmanager or external alertmanager approach, as this was my initial plan and I still don't see a benefit of having both in parallel. 

Therefore everything fine I just wanted make sure that this is now the way to go.